tests/dashboard/system-graphs.test.js

```diff
@@ -3,8 +3,9 @@ const common = require('../common'),
 
```

Comment:
Missing a test if the data updates, and (again if possible) verification that the graphs are not empty